src/postamble.js

```diff
@@ -43,6 +43,9 @@ if (memoryInitializer) {
     memoryInitializer = Module['locateFile'](memoryInitializer);
   } else if (Module['memoryInitializerPrefixURL']) {
     memoryInitializer = Module['memoryInitializerPrefixURL'] + memoryInitializer;
+  } else if (ENVIRONMENT_IS_NODE) {
+    // Look for .mem file in the same folder as script by default
+    memoryInitializer = __dirname + '/' + memoryInitializer;
```

Comment:
If we knew the current directory, though, then we can find the mem init file there. In node we have a mechanism for that, but not in other environments, but they could set `Module['pwd']` when they can, perhaps. Anyhow, if that's not useful, feel free to ignore, I was mostly trying to figure out what 'documentDirectory' was proposed to be.